presto-docs/src/main/sphinx/functions/aggregate.rst

```diff
@@ -564,8 +564,92 @@ To find the `ROC curve <https://en.wikipedia.org/wiki/Receiver_operating_charact
 
     The thresholds are defined as a sequence whose :math:`j`-th entry is the :math:`j`-th smallest threshold.
 
---------------------------
+Differential Entropy Functions
+-------------------------------
+
+The following functions approximate the binary `differential entropy <https://en.wikipedia.org/wiki/Differential_entropy>`_.
+That is, for a random variable :math:`x`, they approximate
+
+.. math ::
+
+    H(x) = - \int x \log_2\left(f(x)\right) dx,
+
+where :math:`f(x)` is the pdf of :math:`x`.
+
+.. function:: differential_entropy(sample_size, x)
+
+    Returns the approximate log-2 differential entropy from a random variable's sample outcomes. Broadly speaking,
```

Comment:
I find it strange to find both `broadly speaking` and `internally creates` next to each other. How about, `This function creates a reservoir...`?